modules/appeals_api/app/swagger/appeals_api/v1/description_with_final_status.md

```diff
@@ -0,0 +1,113 @@
+
+#### Status Simulation
+
+Sandbox test submissions do not progress through the same statuses as in the Production environment. In the lower environments (i.e. Sandbox or Staging), the final status of a submission is `submitted`. In the lower environments, we allow passing a `Status-Simulation` header on the show endpoints so that you can simulate the other statuses.
+
+Statuses can be simulated for all submissions as well as evidence document uploads.
+
+The **submission statuses** available for simulation are the statuses listed in the Submission Statuses table above.
+
+The **evidence upload** statuses available for simulation are the statuses listed in the Evidence Upload Statuses table below (for simulating the status of uploaded evidence documents).
+
+### Evidence Uploads
+
+Our Notice of Disagreement (NOD) and Supplemental Claim (SC) evidence submission endpoints allow a client to upload a document package (documents and metadata) of supporting evidence for their submission by following these steps.
+
+1. Use the POST endpoint `/notice_of_disagreements/evidence_submissions` or `/supplemental_claims/evidence_submissions` to return a JSON service response with the attributes listed below.
+
+- `guid`: An identifier used for subsequent evidence upload status requests (not to be confused with the NOD or SC submission GUID)
+- `location`: A URL to which the actual document package payload can be submitted in the next step. The URL is specific to this upload request, and should not be re-used for subsequent uploads. The URL is valid for 900 seconds (15 minutes) from the time of this response. If the location is not used within 15 minutes, the GUID will expire. Once expired, status checks on the GUID will return a status of `expired`.
+
+1. Client Request: PUT to the location URL returned in step 1.
+
+- Request body should be encoded as binary multipart/form-data, equivalent to that generated by an HTML form submission or using “curl -F…”.
+- No `apikey` authorization header is required for this request, as authorization is embedded in the signed location URL.
+- The metadata.json file uploaded to the location URL with the evidence documents MUST contain all required information.  See example below.
+  -The JSON key for the metadata.json file is "metadata", the initial file is "content". Any subsequent file will be "attachment1", "attachment2", and so forth.
+
+1. The service response will include:
+
+- HTTP status to indicate whether the evidence document upload was successful.
+- ETag header containing an MD5 hash of the submitted payload. This can be compared to the submitted payload to ensure data integrity of the upload.
+
+Example `metadata.json` file:
+```
+{
+    "veteranFirstName": "Jane",
+    "veteranLastName": "Doe",
+    "fileNumber": "012345678",
+    "zipCode": "94402",
+    "source": "Vets.gov",
+    "docType": "316",
+    "finalStatus": false
```

Comment:
`finalStatus` shouldn't have been added here, since this section is explaining the `metadata.json` file that the consumer provides to the endpoint, not the fields that are returned by the endpoint.